conf/screenshot_conf.py

```diff
@@ -0,0 +1 @@
+BS_ENABLE_SCREENSHOTS = False
```

Comment:
@sravantit25 - tiny side note. When you introduce a py file, you should include a doc-string no matter how self-explanatory you think the file is.

Reply:
Yes, Arun, I should have done that. As you said, doc-string needs to be there even if obvious. I will fix it.